spec/models/item_spec.rb

```diff
@@ -1,8 +1,20 @@
 require 'rails_helper'
 
-RSpec.describe Item, :type => :model do
-  let (:order) { Order.create(users_id: 1, status:"ordered", total: 1700)}
-  let (:item) { Item.create(title: "Green Chili Burrito", description:"Amazingly spicey burrito", price:655, image: "image/path")}
+RSpec.describe Item, type: :model do
+  let (:order) { Order.create(
+                              users_id: 1,
+                              status: "ordered",
+                              total: 1700
+                              )
+  }
+  
+  let (:item) { Item.create(
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.